test/MultivariateStats.jl

```diff
@@ -152,5 +152,62 @@ end
     @test mcr ≤ 0.15
 end
 
+@testset "BayesianMulticlassLDA" begin
+    Smarket = dataset("ISLR", "Smarket")
+    X      = selectcols(Smarket, [:Lag1,:Lag2])
+    y      = selectcols(Smarket, :Direction)
+    train  = selectcols(Smarket, :Year) .< 2005
+    test   = .!train
+    Xtrain = selectrows(X, train)
+    ytrain = selectrows(y, train)
+    Xtest  = selectrows(X, test)
+    ytest  = selectrows(y, test)
+
+    BLDA_model = BayesianLDA()
+    fitresult, = fit(BLDA_model, 1, Xtrain, ytrain)
+    class_means, projection_matrix, priors = MLJBase.fitted_params(BLDA_model, fitresult)
+
+    preds = predict(BLDA_model, fitresult, Xtest)
+
+    mce = MLJBase.cross_entropy(preds, ytest) |> mean
+
+    @test 0.685 ≤ mce ≤ 0.690
+
+    @test round.(class_means', sigdigits = 3) == [0.0428 0.0339; -0.0395 -0.0313]
+
+    d = info_dict(BayesianLDA)
+    @test d[:input_scitype] == MLJBase.Table(MLJBase.Continuous)
+    @test d[:target_scitype] == AbstractVector{<:MLJBase.Finite}
+    @test d[:name] == "BayesianLDA"
+end
+
+@testset "BayesianMLDA-2" begin
```

Comment:
this one is not required; I had recoded the other one to match results from Multivariate Stats  but that's now superfluous, the other test is enough